src/v2/Apps/BuyerGuarantee/Routes/BuyerGuaranteeIndex.tsx

```diff
@@ -34,19 +29,9 @@ import { scrollIntoView } from "v2/Utils/scrollHelpers"
 import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
 import { BuyerGuaranteeMeta } from "../Components/BuyerGuaranteeMeta"
 
-interface BuyerGuaranteeIndexProps {
-  headerImage: BuyerGuaranteeIndex_headerImage
-  authenticityImage: BuyerGuaranteeIndex_authenticityImage
-  moneyBackGuaranteeImage: BuyerGuaranteeIndex_moneyBackGuaranteeImage
-  securePaymentImage: BuyerGuaranteeIndex_securePaymentImage
-}
+interface BuyerGuaranteeIndexProps {}
 
-export const BuyerGuaranteeIndex: React.FC<BuyerGuaranteeIndexProps> = ({
-  headerImage,
-  authenticityImage,
-  moneyBackGuaranteeImage,
-  securePaymentImage,
-}) => {
+export const BuyerGuaranteeIndex: React.FC<BuyerGuaranteeIndexProps> = () => {
```

Comment:
```suggestion
export const BuyerGuaranteeIndex: React.FC = () => {
```

#minor The prop definition and line 32 are no longer needed.